cla-backend/cla/config.py

```diff
@@ -77,6 +77,9 @@
 EMAIL_SERVICE = 'SNS'  #: Email service to use for notification emails.
 EMAIL_ON_SIGNATURE_APPROVED = True  #: Whether to email the user when signature has been approved.
 
+# Platform Maintainers
+PLATFORM_MAINTAINERS = ["ddeal@linuxfoundation.org", "makkalot@gmail.com"]
```

Comment:
Can we load this via AWS SSM like the other params?

Reply:
sure 👍